<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>540</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>960</width>
      <height>540</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>960</width>
      <height>540</height>
     </size>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/tealbg.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>220</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255,255,255,1);</string>
    </property>
    <property name="text">
     <string>Pignus Framework Vulnerability Detector</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lab_viere">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>220</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
color:rgba(255,255,255,1);</string>
    </property>
    <property name="text">
     <string>Gathering scan results for viewing</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lab_subtt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>220</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Light</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255,255,255,1);
background-color:rgba(0,0,0,0);
</string>
    </property>
    <property name="text">
     <string>v0.01, Created by t0xic0der</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_one">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,0.75);
border-radius:15px;</string>
    </property>
    <property name="placeholderText">
     <string>Enter the URL for scanning</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_scan_one">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,32,32,0.75);
border-radius:15px;
color: rgba(255,255,255,1);</string>
    </property>
    <property name="text">
     <string>Scan now</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_state">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>680</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Light</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
color:rgba(255,255,255,1);</string>
    </property>
    <property name="text">
     <string>Current status of the results obtained from the scan</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lab_oneur">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>175</y>
      <width>220</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Light</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
color:rgba(255,255,255,1);</string>
    </property>
    <property name="text">
     <string>Enter one URL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_scan_mul">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>435</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,32,32,0.75);
border-radius:15px;
color: rgba(255,255,255,1);</string>
    </property>
    <property name="text">
     <string>Scan now</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_mulur">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>220</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Light</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
color:rgba(255,255,255,1);</string>
    </property>
    <property name="text">
     <string>Enter a text file with multiple URLs</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_mul">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>395</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,0.75);
border-radius:15px;</string>
    </property>
    <property name="placeholderText">
     <string>Enter the URL for scanning</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_orpre">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>220</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
color:rgba(255,255,255,1);</string>
    </property>
    <property name="text">
     <string>OR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lab_mulin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>475</y>
      <width>220</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Light</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
color:rgba(255,255,255,1);</string>
    </property>
    <property name="text">
     <string>The list of URLs should be present in a text file with one URL per line. </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lab_onein">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>220</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Light</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
color:rgba(255,255,255,1);</string>
    </property>
    <property name="text">
     <string>For faster detection of results, add connection protocol to your URL.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="det_boxer">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>680</width>
      <height>410</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Light</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,0.75);
border-radius:15px;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="fwvul.qrc"/>
 </resources>
 <connections/>
</ui>
